Automattic/wp-calypso
# Removed storeFeatures step from stepper

#### Proposed Changes

* This PR removed the storeFeatures step from stepper, since it not used anymore.

#### Testing Instructions

- Go through the site-setup flow (start at `/start`), especially the `sell` intent,  and confirm nothing is broken.
- There shouldn't be any changes from the user perspective!

Closes https://github.com/Automattic/wp-calypso/issues/72157
